qcodes/station.py

```diff
@@ -132,12 +140,13 @@ def __init__(self, *components: Metadatable,
             self.add_component(item, update_snapshot=update_snapshot)
 
         self.use_monitor = use_monitor
-        self.config_file = config_file
 
         self._added_methods: List[str] = []
         self._monitor_parameters: List[Parameter] = []
 
-        self.load_config_file(self.config_file)
+        self.config_file = config_file
```

Comment:
I suggest that we normalize the config_file here such that config file is always a List of strings. 
E.g. if None we should make s
```
if config_file is None:
  self.config_file = [ ]
if isinstance(config_file, str)
   self.config_file = [str,] 
else:
   self.config_file = config_file
```
That way we can write all the other code in the class such that we can always assume that self.config_file is a list